pkg/agent/openflow/framework.go

```diff
@@ -201,7 +201,7 @@ func (f *featurePodConnectivity) getRequiredTables() []*Table {
 			}
 		}
 	}
-	if f.enableTrafficControl {
+	if f.enableTrafficControl || f.enableL7Visibility {
```

Comment:
@wenyingd @tnqn do you think that it makes sense to have this in `featurePodConnectivity`? Should we have a dedicated feature for "visibility" now that we have this table requirement?

Reply:
From feature independency perspective, I would prefer to use a dedicated featuer for "visibility" rather than merging everything into one. A minor concern is, a "feature" in antrea openflow pipeline is expected t implement this interface https://github.com/antrea-io/antrea/blob/main/pkg/agent/openflow/framework.go#L55C6-L55C13, I am not sure if the "visibility" feature has its own flows/groups/meters... If it is an "bare" feature without any OF resources, it would look weird and may introduce difficulty to maintain similar "bare features" in future.

Reply:
Thank you @wenyingd for your suggestion, I am still unclear should we have a dedicated feature for "L7Visibility"?

@tnqn Please help with your suggestion.

Reply:
@tushartathgur Wenying's suggestion would be based on the answer of "if the "visibility" feature has its own flows/groups/meters... If it is an "bare" feature without any OF resources". Could you answer that to help make a more thoughtful suggestion?

From my understanding, the answer is no, there is no dedicated OpenFlow resources for visibility as it just leverages TrafficControl. And having a feature for Visibility but not for TrafficControl may make it more confusing.
Do I understand it correctly?

Reply:
@tnqn , yes your understanding is correct, visibility feature does not have its own OF resources and it leverages TrafficControl for the flows and table.

Reply:
Closing this as it seems the consensus is to _not_ have a dedicated feature